ramp-database/ramp_database/cli.py

```diff
@@ -189,6 +191,52 @@ def add_submission(config, event, team, submission, path):
                                          path)
 
 
+@main.command()
+@click.option("--config", default='config.yml', show_default=True,
+              help='Configuration file YAML format containing the database '
+              'information')
+@click.option("--config_event", required=True,
+              help="""Path to configuration file YAML format
+              containing the database information, eg config.yml""")
+@click.option('--event-name', required=True,
+              help='Name of the event which will be removed.')
+@click.option('--from-disk', is_flag=True, default=False,
+              help="""If True the event and all the related submissions
+                      will be removed from the database and from the disk;
+                      if set to False they will only be removed from
+                      the database""")
+def delete_event(config, config_event, event_name, from_disk):
+    """Delete event."""
+    config = read_config(config)
+
+    # check if the event_config dir exists
+    if not os.path.isfile(config_event) and from_disk:
+        print('cannot find '+config_event)
+        from_disk = False
+
+    # remove event from the database
+    with session_scope(config['sqlalchemy']) as session:
+        # is event in the database?
+        db_event = event_module.get_event(session, event_name)
+        if db_event:
+            event_module.delete_event(session, event_name)
+            if from_disk:
+                # remove submissions dir from disk if it exists
+                path_to_submissions = db_event.path_ramp_submissions
+                if os.path.isfile(path_to_submissions):
```

Comment:
This is weird here. If `path_to_submissions` is a directory, then we should use something else than `isfile`. If it is a list of submissions, I would expect that we loop other the file one by one?